src/relax/transform/fuse_ops.cc

```diff
@@ -900,7 +900,7 @@ class PatternBasedPartitioner : ExprVisitor {
   using Group = GraphPartitioner::Group;
   using GroupMap = OperatorFusor::GroupMap;
   using ExprVisitor::VisitExpr_;
-  using FCheckMatch = backend::PatternRegistryEntryNode::FCheckMatch;
+  using FCheckMatch = runtime::TypedPackedFunc<bool(const Map<DFPattern, Expr>&, const Expr&)>;
```

Comment:
we can remove https://github.com/apache/tvm/blob/03c5f998b37afe41257e2c37b1e95e0ef5357ce8/src/relax/transform/fuse_ops.cc#L43 if it's no longer needed